src/app/common/hooks/use-event-listener.ts

```diff
@@ -43,32 +43,32 @@ const isBrowser = checkIsBrowser();
  *
  * @param event the event name
  * @param handler the event handler function to execute
- * @param doc the dom environment to execute against (defaults to `document`)
+ * @param element the dom environment to execute against (defaults to `document`)
  * @param options the event listener options
  */
 export function useEventListener(
   event: keyof WindowEventMap,
   handler: (event: any) => void,
-  doc: Document | null = isBrowser ? document : null,
+  element: Document | null = isBrowser ? document : null,
```

Comment:
Just FYI I've deprecated this removing `BaseDrawer`, not sure if we need it?

Reply:
If we keep this, should it be moved into the `ui` folder here?

Reply:
Lets just leave it here as I will be deleting it soon. The only place it was used was `baseDrawer` and the default functionality of radix dialog replaces what it was doing.